### PR TITLE
Feat dark mode

### PR DESCRIPTION
**Pull request by kambhani**

### CODE REVIEW

File: .DS_Store
Comment (eduxstad) [inline .DS_Store:1]: Remove this please. You could also add it to the `.gitignore`.

**Comment by eduxstad** [comment]
Can you post a picture of what the site looks like in dark mode? Preferably showing all of the different components.

**Comment by kambhani** [comment]
image
image

File: .DS_Store
Comment (kambhani) [reply .DS_Store:1]: Removed

**Comment by eduxstad** [comment]
Thanks, looks great! I'll get around to deploying this soon, probably this weekend.

**Comment by eduxstad** [comment]
Sorry for the delay, I've deployed these changes now.

**Pull request merged**